tests/deepsparse/image_classification/test_pipelines.py

```diff
@@ -34,8 +34,7 @@
     "zoo_stub,image_size,num_samples",
     [
         (
-            "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/"
-            "pruned85_quant-none-vnni",
+            "zoo:mobilenet_v1-1.0-imagenet-pruned.4block_quantized",
```

Comment:
In general, many models seem to be replaced not by their equivalents, but totally different models.

Reply:
see above